### PR TITLE
i18n(it): Update `guides/components.mdx`

### PR DESCRIPTION
**Pull request by casungo**
#### Description

Updated `guides/components.mdx`

### CODE REVIEW

File: docs/src/content/docs/it/guides/components.mdx [hunk 1]
```diff
@@ -302,3 +278,119 @@ Il codice sopra genera quanto segue nella pagina:
 import importedCode from '/src/env.d.ts?raw';
 
 <Code code={importedCode} lang="ts" title="src/env.d.ts" />
+
+### Albero di File
+
+Utilizza il componente `<FileTree>` per visualizzare la struttura di una directory con icone dei file e directory chiudibili.
+
+Specifica la struttura dei tuoi file e directory con una [lista Markdown non ordinata](https://www.markdownguide.org/basic-syzntax/#unordered-lists) all'interno di `<FileTree>`.
```
Comment (HiDeoo) [inline docs/src/content/docs/it/guides/components.mdx:286]:
```suggestion
Specifica la struttura dei tuoi file e directory con una [lista Markdown non ordinata](https://www.markdownguide.org/basic-syntax/#unordered-lists) all'interno di `<FileTree>`.
```

Small typo leading to a 404

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view) and except one small typo in a link, everything looks perfect to me :thumbsup:

File: docs/src/content/docs/it/guides/components.mdx (hunk 1 above)
Comment (casungo) [reply docs/src/content/docs/it/guides/components.mdx]: Thanks! fixed now :)

**Review by HiDeoo — APPROVED** [review]
Thanks for the fix :rocket:

**Pull request merged**